### PR TITLE
Added recipe for nose-xcover

### PR DESCRIPTION
**Pull request by pmlandwehr**
Adds a recipe for [nose-xcover.](https://pypi.python.org/pypi/nosexcover) Also, pinging @cmheisel in case he would like to be added as a maintainer to the `nosexcover` builder on conda-forge, a library of community-build conda packages. The maintenance burden here is light: all testing and releases are built and deployed automatically with CI. Changes to how it is being built can be controlled by changing the recipe which will be located at https://github.com/conda-forge/nosexcover-feedstock shortly after this PR is finished and merged. If you aren't interested in helping maintain the build, that is entirely fine too!

### CODE REVIEW

**Comment by pmlandwehr** [comment]
(ping @ocefpaf)

File: recipes/nosexcover/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    - nose
+    - coverage >=3.4
+
+test:
+  imports:
+    - nosexcover
+
+about:
+  home: http://github.com/cmheisel/nose-xcover/
+  license: BSD 1-Clause
```
Comment (ocefpaf) [inline recipes/nosexcover/meta.yaml]: @pmlandwehr I believe the actual license is MIT. See https://github.com/cmheisel/nose-xcover/issues/10

The `setup.py` metadata is wrong BTW. (And I don't believe there is a BSD 1-Clause :smile:)

Comment (pmlandwehr) [inline recipes/nosexcover/meta.yaml]: > (And I don't believe there is a BSD 1-Clause 😄)

Aw man...

Comment (pmlandwehr) [inline recipes/nosexcover/meta.yaml]: Fixed the license, added a mssing `entry_point` - other metadata?

**Pull request merged**